src/models/os.ts

```diff
@@ -404,13 +411,14 @@ const getOsModel = function (
 	 */
 	async function getAvailableOsVersions(
 		deviceTypes: string[] | string,
+		options?: { includeDraft?: boolean },
```

Comment:
Maybe this and line 900 should be the options that `_memoizedGetAllOsVersions` expects for its second parameter

Reply:
See: https://github.com/balena-io/balena-sdk/pull/1398#discussion_r1459184748

Reply:
Moreover `_memoizedGetAllOsVersions` is used by both `getAvailableOsVersions` & `getAllOsVersions`.
Till now I couldn't find a nice way to combine them :(